src/Endpoint.sol

```diff
@@ -6,6 +6,7 @@ import "./libraries/PausableOwnable.sol";
 
 abstract contract Endpoint is PausableOwnable {
     function _sendMessage(
+        address token,
```

Comment:
Why are we extending both of these interfaces to include the token address? Since we have the token address available as an immutable in both the endpoints and the manager I don't think we have to pass it around like this?

Reply:
The problem is that `token` isn't available in the `WormholeEndpoint` implementation.
We need to find some way to pass this token address to the `WormholeEndpoint`, so I opted for adding the token to the internal `_sendMessage` method.

Perhaps there's another way to do this where we expose a virtual `token()` method in the abstract Endpoint contract that the `WormholeEndpoint` will call, and this'll work when all the contracts are put together?

Reply:
But we're now adding it as an immutable? https://github.com/wormhole-foundation/example-native-token-transfers/blob/quoter/src/EndpointStandalone.sol#L25

And it's already accessible in the Manager so for both cases it's available no?